src/Main.js

```diff
@@ -7,7 +7,7 @@ class Main extends Component {
     return (
       <>
         <HornedBeast title={'Name'} imgURL={name} decriptio={'Descriptive block'} />
-        <HornedBeast title={'Scratchy'}
+        <HornedBeast title={'Title'}
       </>
     )
   }
```

Comment:
Here I see that you are importing HornedBeast component but it looks like you:

- need to import in the `HornedBeast` component into Main (if you want Main to consume it).

- doublecheck that your imports for images are working properly as you will want to import an image from a certain directory in your project. I couldn't find the images in your project directory or the HornedBeast component.

I would get with a TA to get you up and running on this if ya can and if I missed something here, let me know :) 👍